lib/src/component/abstract_transition.dart

```diff
@@ -281,8 +281,16 @@ abstract class AbstractTransitionComponent<T extends AbstractTransitionProps, S
     }
   }
 
+  var _isMounted = false;
+
+  @override
+  void componentDidMount() {
+    _isMounted = true;
```

Comment:
Doesn't this fall back to the antipattern of tracking mounts? 

https://facebook.github.io/react/blog/2015/12/16/ismounted-antipattern.html